Cargo.toml

```diff
@@ -155,10 +158,13 @@ cli = [
    "dep:opentelemetry-system-metrics"
 ]
 
+# Feature Flag to enable Apollo Tracing.
+apollo-tracing = ["dep:async-graphql-extension-apollo-tracing"]
```

Comment:
why as separate feature and not like part of `cli` above?

Reply:
since we encountered multiple issues with the library, it felt like keeping it as a separate feature flag will allow anyone to disable the library if it causes any other issue